fetch/nosniff/resources/image.py

```diff
@@ -1,22 +1,24 @@
 import os.path
 
+from wptserve.utils import isomorphic_decode
+
 def main(request, response):
-    type = request.GET.first("type", None)
+    type = request.GET.first(b"type", None)
 
-    if type != None and "svg" in type:
-        filename = "green-96x96.svg"
+    if type != None and b"svg" in type:
+        filename = u"green-96x96.svg"
     else:
-        filename = "blue96x96.png"
+        filename = u"blue96x96.png"
 
-    path = os.path.join(os.path.dirname(__file__), "../../../images", filename)
-    body = open(path, "rb").read()
+    path = os.path.join(os.path.dirname(isomorphic_decode(__file__)), u"../../../images", filename)
```

Comment:
Same question about this instance of `isomorphic_decode(__file__)`.

Reply:
same reason as above.